### DISCUSSION TITLE
qdarnt cloud GCP hosted cluster not available for Tokyo region

### DISCUSSION THREAD

**Discussion by manishkr-z** [qdrant/qdrant#3541] [Q&A]
![image](https://github.com/qdrant/qdrant/assets/129715166/407edb31-09af-43cc-9e4d-cdd1872df460)

Does Qdrant have plans to increase the availability regions in gcp ?

**Comment by elbart** [comment]
@manishkr-z Thank you for your request. We are frequently monitoring demand for individual regions. Would you tell us a bit more about your plans with using qdrant to understand the potential demand?
